Logger/LogFactory.cs

```diff
@@ -1,11 +1,36 @@
-﻿namespace Logger
+﻿using System;
+
+namespace Logger
 {
     public class LogFactory
     {
+        private string path;
```

Comment:
Fields should be underscore PascalCase
```suggestion
        private string _Path;
```